horovod/common/operations.cc

```diff
@@ -865,6 +899,21 @@ void PerformOperation(TensorTable& tensor_table, MPIResponse response) {
 
         ACTIVITY_END_ALL(entries, timeline)
       }
+#elif HOROVOD_GPU_ALLREDUCE == 'D'
+      if (!horovod_global.ddl_initialized) {
+        // Initialize DDL
+        auto ddl_options = std::getenv("DDL_OPTIONS");
```

Comment:
Can you give an example of DDL_OPTIONS?  Are they system-wide?

Reply:
They are setup by ddlrun (or you can set it up yourself too). I would refer to the DDL documentation to get the specific about DDL_OPTIONS.